pkg/k8s/kubernetes.go

```
@@ -449,6 +487,28 @@ func BuildSUT(client *kubernetes.Clientset, s *config.PerfScenarios) error {
 	return nil
 }
 
+// Extract the UDN Ip address of a pod from the annotations - Support only ipv4
+func ExtractUdnIp(s config.PerfScenarios) string {
+	podNetworksJson := s.Server.Items[0].Annotations["k8s.ovn.org/pod-networks"]
+	var podNetworks map[string]interface{}
+	err := json.Unmarshal([]byte(podNetworksJson), &podNetworks)
+	if err != nil {
+		log.Warn("Not able to extract UDN ip", err)
```

if we cannot extract the data, should we return an error from here?